Cargo.toml

```diff
@@ -1,13 +1,34 @@
 [package]
-authors = ["Nick Fitzgerald <fitzgen@gmail.com>"]
+authors = ["The Starling Project Developers"]
+description = "The project that will henceforth be known as `starling`."
```

Comment:
Forgot to mention this earlier: let's change this to "The Starling JS runtime". Not an ideal description, but I can't come up with anything better that's also concise right now. Proposals welcome!